pkg/generate/app/templatelookup.go

```diff
@@ -57,9 +61,13 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
 						Score:       score,
 						Template:    template,
 					})
+					break
```

Comment:
is that what you really want?  I guess i don't understand what the "levels" (multiple terms) are here...  but i would think if term1 had a lousy match, you'd still want to check to see if term2 had a good/better match.